### PR TITLE
README: added "Our Contributors"

### PR DESCRIPTION
**Pull request by TheMimikyu**
Opening a pull request on MapComplete
=====================================

Hey! Thanks for opening a pull request on Mapcomplete! This probably means you want to add a new theme - if so, please
follow the checklist below. If this pull request is for some other issue, please ignore the template.

Adding your new theme
---------------------

Thanks for taking the time to create your theme and to add it to the main repository!

To making merging smooth, please make sure that each of the following conditions is met:

- [x] The codebase is GPL-licensed. By opening a pull request, the new theme will be GPL too
- [x] All images are included in the pull request and no images are loaded from an external service (e.g. Wikipedia)
- [x] The [guidelines on how to make your own theme](https://github.com/pietervdvn/MapComplete/blob/master/Docs/Making_Your_Own_Theme.md)
  are read and followed

### CODE REVIEW

**Comment by pietervdvn** [comment]
Hi,

What is the privacy policy of the linked service?

Do they track everyone who is simply visiting

File: README.md [hunk 1]
```diff
@@ -180,6 +180,13 @@ changeset is probably closed and we open a new changeset.
 Note that changesets are closed automatically after one hour of inactivity, so we don't have to worry about closing
 them.
 
+# Our Contributors ⭐
+
+<a href="https://github.com/pietervdvn/MapComplete
```
Comment (pietervdvn) [inline README.md:185]: There is a newline here, breaking the link and the formatting. This must be fixed before merging this in

**Review by pietervdvn — CHANGES_REQUESTED** [review]
Fix formatting + link privacy policy

File: README.md (hunk 1 above)
Comment (TheMimikyu) [reply README.md:185]: Apologies from my side, I couldn't find the service's privacy policy. 
I have fixed the formatting in my latest commit.

**Pull request closed without merging**